core/templates/pages/classroom-page/classroom-page.component.html

```diff
@@ -99,6 +99,9 @@ <h2 translate="I18N_CLASSROOM_PAGE_TOPICS_COVERED"></h2>
     margin-top: -10vw;
     width: 25vw;
   }
+  .oppia-classroom-page-subheading-text {
```

Comment:
Please add a prefix selector for this. All CSS rules should start with the top-level CSS class of the directive, so that they don't affect other elements outside it.